project/.github/workflows/release.yml.jinja

```
@@ -0,0 +1,25 @@
+name: release
+
+on: push
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Fetch all tags
+      run: git fetch --depth=1 --tags
+    - name: Setup Python
+      uses: actions/setup-python@v4
+    - name: Install git-changelog
+      run: pip install git-changelog
```

```suggestion
      run: pipx install git-changelog
```
I guess that also works.

---

I guess it makes it a bit more safe (less risk of dependency conflicts in the `pip` env) :+1: